src/iOS/Xamarin.iOS/Samples/Data/RasterLayerGeoPackage/RasterLayerGeoPackage.cs

```diff
@@ -44,48 +45,55 @@ public override void ViewDidLoad()
 
         public override void ViewDidLayoutSubviews()
         {
-            base.ViewDidLayoutSubviews();
-
-            // Update the UI to account for new layout
-            _myMapView.Frame = new CoreGraphics.CGRect(0, 0, View.Bounds.Width, View.Bounds.Height);
+            try
+            {
+                nfloat topMargin = NavigationController.NavigationBar.Frame.Height + UIApplication.SharedApplication.StatusBarFrame.Height;
+
+                // Reposition controls.
+                _myMapView.Frame = new CoreGraphics.CGRect(0, 0, View.Bounds.Width, View.Bounds.Height);
+                _myMapView.ViewInsets = new UIEdgeInsets(topMargin, 0, 0, 0);
+
+                base.ViewDidLayoutSubviews();
```

Comment:
It doesn't matter. I've changed (as far as I know) all instances to follow a pattern. Ultimately, all of this code is an anti-pattern; we should be using AutoLayout and layout controls rather than manual frame-based positioning.